client/src/api/getEvents.js

```diff
@@ -0,0 +1,10 @@
+export const getEvents = async () => {
+  const response = await fetch("/api/events");
+  if (!response.ok) {
+    throw response;
+  }
+
+  const result = await response.json();
+  console.log(result);
```

Comment:
Do you still need this console.log? If not, you could remove it

Reply:
deleted